graveyard.json

```diff
@@ -1494,5 +1495,13 @@
     "link": "https://en.wikipedia.org/wiki/Field_Trip_(application)",
     "name": "Field Trip",
     "type": "app"
+  },
+  {
+    "dateClose": "2019-07-05",
+    "dateOpen": "2018-08-01",
+    "description": "The Dragonfly search engine was reportedly designed to link users' phone numbers to their search queries and censor websites such as Wikipedia and those that publish information about freedom of speech,[10] human rights, democracy, religion, and other issues considered sensitive by the Chinese government.",
```

Comment:
Please review and shorten the description to a single sentence. The first sentence of the Wikipedia entry can be adjusted to fit easily and still provide enough relevant information about the product. Here's an example:

```
Dragonfly was an Internet search engine prototype that was designed to be compatible with China's state censorship provisions.
```